### PR TITLE
feat: add created proposals box and update profile styles

### PR DESCRIPTION
**Pull request by andyesp**
image

image

image

### CODE REVIEW

File: src/components/Profile/ProposalsCreatedBox.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    hasMoreProposals,
+  } = usePaginatedProposals({
+    user: address.toLowerCase(),
+    itemsPerPage: PROPOSALS_PER_PAGE,
+  })
+
+  return (
+    <Container className="ProposalsCreatedBox">
+      <ProfileBox title={t('page.profile.created_proposals.title')}>
+        {isLoading && <SkeletonBars amount={proposals.length || 3} height={89} />}
```
Comment (ncomerci) [inline src/components/Profile/ProposalsCreatedBox.tsx:37]: I would say `<SkeletonBars amount={proposals.length || PROPOSALS_PER_PAGE} ... />`

File: src/components/Profile/VpDelegationBox.css [hunk 2]
```diff
@@ -2,6 +2,7 @@
   padding: 12px 12px 0 !important;
 }
 
+/* TODO: Move this to VotedProposalsBox.css */
```
Comment (ncomerci) [inline src/components/Profile/VpDelegationBox.css:5]: Reminder hehe

**Review by ncomerci — APPROVED** [review]
🥳

File: src/hooks/usePaginatedProposals.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from 'react'
+
+import { ProposalAttributes } from '../entities/Proposal/types'
+
+import useProposals, { UseProposalsFilter } from './useProposals'
+
+export default function usePaginatedProposals(filter: Partial<UseProposalsFilter> = {}) {
```
Comment (1emu) [inline src/hooks/usePaginatedProposals.ts:7]: 🥷

**Review by 1emu — APPROVED** [review]
✨ 💪

**Pull request merged**